
Property changes on: .
___________________________________________________________________
Added: svn:ignore
   + bin
obj



Property changes on: dbg
___________________________________________________________________
Added: svn:ignore
   + bin
obj



Property changes on: symbols\pdb
___________________________________________________________________
Added: svn:ignore
   + obj


Index: symbols/pdb/Microsoft.Cci.Pdb/PdbFunction.cs
===================================================================
--- symbols/pdb/Microsoft.Cci.Pdb/PdbFunction.cs	(revision 709)
+++ symbols/pdb/Microsoft.Cci.Pdb/PdbFunction.cs	(working copy)
@@ -208,8 +208,10 @@
                                     proc.parent, proc.next);
       }
       if (proc.dbgStart != 0 || proc.dbgEnd != 0) {
-        throw new PdbDebugException("Warning DBG start={0}, end={1}",
-                                    proc.dbgStart, proc.dbgEnd);
+        //Commented out by Ðonny because this exception didn't make sense for me and prevented me from loading one C++/CLI-generated PDB file.
+        //See https://groups.google.com/group/mono-cecil/browse_thread/thread/2098d60a2cadb64
+        //throw new PdbDebugException("Warning DBG start={0}, end={1}",
+        //                            proc.dbgStart, proc.dbgEnd);
       }
 
       int constantCount;
